model-optimizer/extensions/middle/FusedBatchNormTraining.py

```diff
@@ -58,7 +58,7 @@ def replace_pattern(self, graph: Graph, match: dict):
 
         input_rank = len(node.in_port(0).data.get_shape())
         rng = create_op_with_const_inputs(graph, Range,
-                                          {0: int64_array(2), 1: int64_array(input_rank), 2: int64_array(1)},
+                                          {0: int64_array(1), 1: int64_array(input_rank - 1), 2: int64_array(1)},
```

Comment:
Please, add new or update existed tests for this functionality in model-optimizer/unit_tests/extensions/middle/FusedBatchNormTraining_test.py